.gitignore

```diff
@@ -35,6 +35,7 @@ googletest/m4/ltoptions.m4
 googletest/m4/ltsugar.m4
 googletest/m4/ltversion.m4
 googletest/m4/lt~obsolete.m4
+googlemock/m4
```

Comment:
Some things are missing from .gitignore. Here are a full list of files (in sorted order) that should be added:

```
$ git diff .gitignore 
diff --git a/.gitignore b/.gitignore
index 6209bf66..5afc957c 100644
--- a/.gitignore
+++ b/.gitignore
@@ -24,17 +24,54 @@ Win32-Release/
 x64-Debug/
 x64-Release/
 
+# Ignore binaries built by tests in tree.
+googlemock/gmock-actions_test
+googlemock/gmock-cardinalities_test
+googlemock/gmock-function-mocker_test
+googlemock/gmock-generated-actions_test
+googlemock/gmock-generated-function-mockers_test
+googlemock/gmock-generated-matchers_test
+googlemock/gmock-internal-utils_test
+googlemock/gmock-matchers_test
+googlemock/gmock-more-actions_no_exception_test
+googlemock/gmock-more-actions_test
+googlemock/gmock-nice-strict_test
+googlemock/gmock-port_test
+googlemock/gmock-spec-builders_test
+googlemock/gmock_ex_test
+googlemock/gmock_leak_test_
+googlemock/gmock_link_test
+googlemock/gmock_no_rtti_test
+googlemock/gmock_output_test_
+googlemock/gmock_stress_test
+googlemock/gmock_test
+googlemock/shared_gmock_test_
+
 # Ignore autoconf / automake files
+*.lo
+*.o
+.deps
+.dirstamp
+.libs
 Makefile.in
 aclocal.m4
-configure
-build-aux/
 autom4te.cache/
+build-aux/
+configure
+config.log
+config.status
+googlemock/lib/
+googlemock/libtool
+googlemock/m4/
+googlemock/scripts/gmock-config
+googletest/libtool
+googletest/lib/
 googletest/m4/libtool.m4
 googletest/m4/ltoptions.m4
 googletest/m4/ltsugar.m4
 googletest/m4/ltversion.m4
 googletest/m4/lt~obsolete.m4
+googletest/scripts/gtest-config
 
 # Ignore generated directories.
 googlemock/fused-src/
```